pkg/plugins/runtime/k8s/plugin.go

```diff
@@ -299,13 +296,6 @@ func addValidators(mgr kube_ctrl.Manager, rt core_runtime.Runtime, converter k8s
 	return nil
 }
 
-type upstreamValidatorHandler struct {
-}
-
-func (g *upstreamValidatorHandler) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
-	gapi_admission.ServeHTTP(writer, request)
-}
-
```

Comment:
How do we plug it now?

Reply:
The upstream webhook is included in the [install resources](https://gateway-api.sigs.k8s.io/guides/getting-started/#installing-gateway-api)

Reply:
I think it's probably worth bundling these resources ourselves with `kumactl` but until that's done, I don't see why we should have it running twice.